packages/go_router_builder/example/lib/readme_excerpts.dart

```diff
@@ -0,0 +1,403 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs, unreachable_from_main, avoid_print, unused_element, unused_local_variable, directives_ordering
+
+import 'package:flutter/material.dart';
+import 'shared/data.dart';
+// #docregion import
+import 'package:go_router/go_router.dart';
+
+part 'readme_excerpts.g.dart';
+// #enddocregion import
+
+void otherDoc(BuildContext context) {
+  // #docregion GoRoute
+  GoRoute(
+    path: ':familyId',
+    builder: (BuildContext context, GoRouterState state) {
+      // require the familyId to be present and be an integer
```

Comment:
Comments, especially for docs, should be capitalized and punctuated like a sentence.